subt/ros/robot/launch/freyja.launch

```diff
@@ -184,9 +184,16 @@
     <rosparam param="imu0_config">[false, false, false,
                                    true,  true,  true,
                                    false, false, false,
-                                   true,  true,  true,
+                                   false,  false,  false,
```

Comment:
It turns out I misunderstood the IMU data and this change from three trues to three falses shouldn't be there. The data we receive are a combination of a magnetometer and a gyroscope, which means that the rotation and rotation velocity measurements are independent and we can use them both.

Reply:
OK - reverted. Let me know when you want to do another run (I will update this branch before it).